plugins/org.wso2.developerstudio.eclipse.artifact.registry/src/org/wso2/developerstudio/eclipse/artifact/registry/ui/wizard/RegistryResourceCreationWizard.java

```diff
@@ -67,27 +67,28 @@
 import org.wso2.developerstudio.eclipse.platform.ui.wizard.pages.ProjectOptionsPage;
 import org.wso2.developerstudio.eclipse.utils.file.FileUtils;
 
-public class RegistryResourceCreationWizard extends AbstractWSO2ProjectCreationWizard {
+public class RegistryResourceCreationWizard extends

```

Comment:
Since you haven't done any changes for this class, do not add this class file in the commit.